src/core/hle/libraries/libkernel/file_system.h

```diff
@@ -7,11 +7,37 @@ class SymbolsResolver;
 }
 
 namespace Core::Libraries::LibKernel {
+constexpr int SCE_MAX_PATH = 255;
 
-int PS4_SYSV_ABI sceKernelOpen(const char *path, int flags, /* SceKernelMode*/ u16 mode);
+struct SceKernelDirent {
+    uint32_t d_fileno;             /* file number of entry */
+    uint16_t d_reclen;             /* length of this record */
+    uint8_t d_type;                /* file type, see below */
+    uint8_t d_namlen;              /* length of string in d_name */
```

Comment:
```suggestion
    u32 d_fileno;              /* file number of entry */
    u16 d_reclen;             /* length of this record */
    u8 d_type;                  /* file type, see below */
    u8 d_namlen;             /* length of string in d_name */
```
Use common types for consistency